src/main/java/net/snowflake/client/core/SFJsonResultSet.java

```diff
@@ -32,6 +32,12 @@
 
   TimeZone timeZone;
 
+  // Precision of maximum long value in Java (2^63-1). Precision is 19
+  private static final int LONG_PRECISION = 19;
+
+  BigDecimal MAX_LONG_VAL = new BigDecimal(Long.MAX_VALUE);
```

Comment:
why not private static final? We don't need to initialize this constants for each instance.

Reply:
ah, good point. Done.